flink-dist/src/main/flink-bin/mesos-bin/mesos-jobmanager.sh

```diff
@@ -26,6 +26,19 @@ bin=$(cd "${bin}" || exit; pwd)
 # get Flink config
 . "${bin}"/config.sh
 
+java_utils_output=$(runBashJavaUtilsCmd GET_JM_RESOURCE_PARAMS "${FLINK_CONF_DIR}" "${FLINK_BIN_DIR}/bash-java-utils.jar:$(findFlinkDistJar)" "${ARGS[@]}")
+logging_output=$(extractLoggingOutputs "${java_utils_output}")
```

Comment:
is there a way to deduplicate this with `taskmanager.sh`? It's _pretty much_ the same thing.